src/raps/actions/unlock.ts

```diff
@@ -145,7 +151,8 @@ const unlock = async (
       accountAddress,
       assetAddress,
       contractAddress,
-      chainId
+      chainId,
+      contractAddress === RAINBOW_ROUTER_CONTRACT_ADDRESS
```

Comment:
I'd move this into an allowsPermit const so you can easily tell what's this param about

Reply:
done